lib/measured/unit.rb

```diff
@@ -0,0 +1,51 @@
+class Measured::Unit
+  include Comparable
+
+  def initialize(name, aliases: [], value: nil)
+    @name = name.to_s
+    @names = ([@name] + aliases.map{|n| n.to_s }).sort
+
+    @conversion_amount, @conversion_unit = parse_value(value) if value
+  end
+
+  attr_reader :name, :names, :conversion_amount, :conversion_unit
```

Comment:
order doesn't really matter, but I usually see attr_readers at the top of the class. Thoughts?